src/main/resources/application.properties

```diff
@@ -14,5 +14,7 @@ management.endpoints.web.base-path=/
 management.endpoints.web.path-mapping.health=healthw
 
 server.error.include-stacktrace=never
+server.shutdown=graceful
 
```

Comment:
super minor but if these two new lines could be grouped together since they're related, that would be cool

also github linter is complaining about no newline at end (sorry not related to your PR)

Reply:
good catch, i've grouped them up